rest/config.go

```diff
@@ -700,15 +700,44 @@ func (dbConfig *DbConfig) validateVersion(ctx context.Context, isEnterpriseEditi
 		}
 	}
 
-	if validateOIDCConfig && dbConfig.OIDCConfig != nil {
-		for name, provider := range dbConfig.OIDCConfig.Providers {
-			_, _, err := provider.DiscoverConfig(ctx)
-			if err != nil {
-				multiError = multiError.Append(fmt.Errorf("failed to validate OIDC configuration for %s: %w", name, err))
+	seenIssuers := base.Set{}
+	if dbConfig.OIDCConfig != nil {
+		validProviders := len(dbConfig.OIDCConfig.Providers)
+		for name, oidc := range dbConfig.OIDCConfig.Providers {
+			if oidc.Issuer == "" || base.StringDefault(oidc.ClientID, "") == "" {
+				// TODO: rather than being an error, this skips the current provider to avoid a BWC issue (previously valid
```

Comment:
```suggestion
				// TODO: rather than being an error, this skips the current provider to avoid a backwards compatibility issue (previously valid
```